Expensify/App
# remove expensify card from bank icons

### Details

Originally, we were showing all imported credit cards, bank accounts, and payment cards in the payments page of the app. I made [this change](https://github.com/Expensify/App/pull/5216) in order to update the icon that was being used when an Expensify Card was one of the imported cards. 

With [this PR](https://github.com/Expensify/App/pull/5212) from @stitesExpensify, we're no longer showing imported credit cards, we're now only showing bank accounts and payment cards. Bank accounts show the bank logo and payment cards always show the default credit card icon. Because of this, an Expensify/Expensify Card logo will never be needed since Expensify isn't a possible option as an imported bank account. 

So this PR effectively reverts what https://github.com/Expensify/App/pull/5216 did since https://github.com/Expensify/App/pull/5212 made it obsolete. 

### Tests

1. Open the Payments page on an account that has a at least one bank account or payment card
2. Make sure the bank accounts/payment cards are showing. A bank should have the bank icon and a payment card should have the credit card icon

### QA Steps

1. Open the Payments page on an account that has a payment card and at least one imported bank account
2. Make sure the bank accounts are showing with their icon and the payment card is showing with the credit card icon

### Tested On
I tested on web as you can see here. There's nothing really to test, since the icon being removed will never actually show. 

![image](https://user-images.githubusercontent.com/1371996/133901767-bb7355e0-cf5a-4312-aff8-73ac39f3e211.png)
